platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -151,11 +140,16 @@ message RosterEntry {
     bytes gossip_ca_certificate = 3;
 
     /**
-     * An elliptic curve public signing key.<br/>
-     * This contains the _long term_ public key for this node.
+     * An ALT_BN128 elliptic curve public encryption key.<br/>
+     * <p>
+     * The elliptic curve type may change in the future. For example <br/>
+     * If the Ethereum ecosystem creates precompiles for BLS12_381,
```

Comment:
Minor nit: `If` capitalization at line 146 is wrong. Also, a comma seems to be missing before it.